Задания с 18.02.2022/Task2/Task2/BWT.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+public class BWT
+{
+    private static string leftShift(string line, int count)
+    {
+        return line.Substring(count) + line.Substring(0, count);
+    }
+
+    public static string Direct(string line, ref int endPosition)
+    {
+        string[] cyclicShifts = new string[line.Length];
```

Comment:
```suggestion
        var cyclicShifts = new string[line.Length];
```
В C# почти никогда не надо писать один и тот же тип дважды в одной строчке